scripts/_redirects

```diff
@@ -154,27 +154,27 @@
 
 # Build Section
 
-/docs/build-network-overview    /build/build-network-overview    301
+/docs/build-network-overview    https://docs.polkadot.com/polkadot-protocol/basics/networks    301
 /docs/build-smart-contracts    /build/build-smart-contracts    301
-/docs/build-hackathon    /build/build-hackathon    301
+/docs/build-hackathon    /build/build-hackathon   301
 /docs/build-open-source    /build/build-open-source    301
-/docs/build-guides-coretime-start    /build/build-guides-coretime-start    301
+/docs/build-guides-coretime-start    https://docs.polkadot.com/tutorials/polkadot-sdk/parachains/zero-to-hero    301
 /docs/build-storage    /build/build-client-side/#decentralized-storage-options    301
-/docs/build-dapp    /build/build-client-side   301
+/docs/build-dapp    https://docs.polkadot.com/develop/#application-developers    301
 /docs/build-pdk    /build/build-parachains   301
-/docs/build-light-clients    /build/build-light-clients     301
-/docs/build-node-interaction    /build/build-node-interaction    301
-/docs/build-client-side    /build/build-client-side    301
+/docs/build-light-clients    https://docs.polkadot.com/develop/toolkit    301
```

Comment:
This is not appropriate. Till there is a doc on Substrate Connect or Smoldot on Polkadot docs, we keep the light clients page as is